app/assets/javascripts/components/CartedProducts.jsx

```diff
@@ -79,7 +91,9 @@ var CartedProducts = React.createClass({
             )}
           </tbody>
         </table>
+        <hr/>
         <div>
+          <p><b>Total: </b>{(this.state.cart_total * 0.01).toLocaleString("en-US", {style: "currency", currency: "USD", minimumFractionDigits: 2})}</p>
```

Comment:
@DillonStreator I noticed that as well. It's because all the calls to the `Stripe API` ...I'm actually working on that and I'll submit a PR tonight or tomorrow to address that. Since you added `price` and `name` to the `CartedProduct` model we can really cut down on when we need to query from Stripe.